payments/views.py

```diff
@@ -102,6 +102,12 @@ def post(self, request, *args, **kwargs):
             profile.money += amount_received
             profile.save()
         elif event.type == 'payment_intent.payment_failed':
-            pass
-        # Passed signature verification
+            return render(request, template_name='payment-failure.html')
```

Comment:
to tak nie zadziala - plis wycofaj zmiany w widoku obslugujacym webhooki bo wyjdzie niepotrzebny konflikt, a odnosniki i redirecty do tej podstrony damy w momencie kiedy calosc bedzie dzialac